release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -679,6 +679,18 @@ You cannot upgrade {op-system-base} 7 compute machines to {op-system-base} 8. Yo
 
 * Previously, editing a machine specification on {rh-openstack-first} would cause {product-title} to attempt to delete and recreate the machine. As a result, this caused an unrecoverable loss of the node it was hosting. With this fix, any edits made to the machine specification after creation are ignored. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1962066[*BZ#1962066*])
 
+* Previously, the check to ensure that the AWS machine was not updated before requeueing was removed. Consequently, problems arose when the AWS machine's virtual machine had been removed, but its object was still available. In the event that this happened, the AWS machine would requeue in an infinite loop and could not be deleted or updated. This update restores the check that was used to ensure that the AWS machine was not updated before requeueing. As a result, machines no longer requeue if they have been updated. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2007802[*BZ#2007802*])
+
+* Previously, modifying a selector changed the list of machines that the MachineSet observed. As a result, leaks could occur because the MachineSet lost track of machines it had already created. This update ensures that the selector is immutable once created. As a result, the MachineSet now lists the correct machines. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2005052[*BZ#2005052*])
```

Comment:
s/MachineSet/machine set
```suggestion
* Previously, modifying a selector changed the list of machines that a machine set observed. As a result, leaks could occur because the machine set lost track of machines it had already created. This update ensures that the selector is immutable once created. As a result, machine sets now lists the correct machines. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2005052[*BZ#2005052*])
```